backend/dataall/modules/maintenance/api/types.py

```diff
@@ -0,0 +1,8 @@
+"""Defines the object types of the Maintenance activity"""
+
+from dataall.base.api import gql
+
+
+Maintenance = gql.ObjectType(
+    name='Maintenance', fields=[gql.Field(name='status', type=gql.String), gql.Field(name='mode', type=gql.String)]
```

Comment:
nit: `status` should be non-nullable

Reply:
Thanks!!. Made updates in the next commit